arrow/formatter.py

```diff
@@ -120,3 +120,75 @@ def _format_token(self, dt, token):
         if token == "W":
             year, week, day = dt.isocalendar()
             return "{}-W{:02d}-{}".format(year, week, day)
+
+    # COOKIE FORMAT: 'Thursday, 25-Dec-1975 14:15:16 EST'
+    def _format_cookie(self, dt):
```

Comment:
So these functions are meant to be used like this:

```
>>> ar = arrow.get("1975-12-25T14:15:16")
<Arrow [1975-12-25T14:15:16+00:00]>
>>> ar.format_cookie()
'Thursday, 25-Dec-1975 14:15:16 EST'
```

We need to pass in `cls` rather than `self` and we should remove the leading underscores.